lib/resources/nginx.rb

```diff
@@ -34,7 +34,7 @@ def initialize(nginx_path = '/usr/sbin/nginx')
       read_content
     end
 
-    %w{compiler_info error_log_path http_client_body_temp_path http_fastcgi_temp_path http_log_path http_proxy_temp_path http_scgi_temp_path http_uwsgi_temp_path lock_path modules_path openssl_version prefix sbin_path service support_info version}.each do |property|
+    %w{error_log_path http_client_body_temp_path http_fastcgi_temp_path http_log_path http_proxy_temp_path http_scgi_temp_path http_uwsgi_temp_path lock_path modules_path prefix sbin_path service version}.each do |property|
```

Comment:
Same here, why are removing these properties?

Reply:
compiler info is defined as a method on line 48; openssl_version on 43, and support_info on 53.